src/transform/class/index.js

```diff
@@ -94,6 +96,15 @@ export default function(ast, logger) {
           });
         }
       }
+      else if ((m = inheritance.process(node, parent))) {
+        if (potentialClasses[m.className]) {
+          potentialClasses[m.className].superClass = m.superClass;
+          potentialClasses[m.className].replacements = [
+            ...potentialClasses[m.className].replacements,
+            ...m.replacements
+          ];
```

Comment:
Accessing the fields of `PotentialClass` directly breaks encapsulation. We should add a method instead.